fastlane_core/spec/build_watcher_spec.rb

```diff
@@ -2,111 +2,22 @@
 
 describe FastlaneCore::BuildWatcher do
   context '.wait_for_build_processing_to_be_complete' do
-    let(:processing_build) do
```

Comment:
why is this (and following removed lines) not needed any more?

Reply:
This was a duplicate that should have deleted a while ago when build watcher changed over from using `Spaceship::TestFlight` to `Spaceship::ConnectAPI`